cinema/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    description = serializers.CharField()
+    duration = serializers.IntegerField()
+
+    def create(self, validated_data: dict) -> Movie:
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):

```

Comment:
In the create method, you used annotations, but in the update method, you didn't. You should either use them everywhere or not use them at all.